File: archives/python/2857.py
import sys
input = sys.stdin.readline

cnt = 0
FBI = [False] * 5
for i in range(5):
    word = input().strip()
    if "FBI" in word:
        FBI[i] = True
        cnt += 1
if cnt == 0:
    print("HE GOT AWAY!")
else:
    print(*[idx + 1 for idx, x in enumerate(FBI) if x])

File: archives/cpp/2857.cpp
#include <bits/stdc++.h>
#define FOR(i, start, end) for (int i = start; i < end; i++)
#define endl "\n"
#define INT_INF 2147483647
#define MOD 1
using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    bool fbi = false;
    FOR(i, 1, 6){
        string word; cin >> word;
        if (word.find("FBI") != -1) {
            cout << i << ' ';
            fbi = true;
        }
    }
    if (!fbi) {
        cout << "HE GOT AWAY!";
    }

    return 0;
}